docs/templates/pathTemplate.yml

```diff
@@ -0,0 +1,605 @@
+    #
+    # - Add an owner to a bucket
+    # - Create a bucket
+    # 
+    # For more guidance, see https://developers.google.com/style/headings?hl=en#heading-and-title-text
+    summary: Create a task
+
+    # Operation description
+    #
+    # 1. In the first sentence, start with a verb
+    #   (Adds, Creates, Deletes, Lists, Updates, Returns)
+    #    and briefly state in present tense what action the operation performs
+    #    --for example:
+    #
+    #    - Returns a task.
+    #    - Adds a new bucket to the organization and returns the bucket.   
+    #    - Updates the authorization status and returns the active or inactive
+    #      authorization.
+    #
+    #    For more help, see https://developers.google.com/style/api-reference-comments?hl=en#description).
+    #
+    # 2. For deprecated operations, set the `deprecated` Operation field to
+    #    `true` and add a link here to the new operation--for example:
+    #
+    #    - **Deprecated**.
+    #      Use [`PATCH /api/v2/tasks/{taskID}`](#operation/PatchTasksID) instead.
+    # 
+    # 3. Describe the following:
+    #   - Why and when to use this Operation
+    #   - Idempotency
+    #   - Side effects
+    #   - Requirements that are "hidden" or not obvious from the requestBody or
+    #   - parameter definitions
+    #   - Assumptions data sources, formats, I/O, or the state of a resource
+    #   - Nuances in requests, response codes, or payloads.
+    #
+    # 4. Describe unique relationships to other InfluxDB resources or concepts,
+    #    especially if they're not obvious--for example:
+
+    #    - Use `POST /api/v2/authorizations` to create InfluxDB API tokens.
+    # 
+    # #### Required permissions
+    #
+    # - List permissions required for the operation.
+    #
+    # #### Rate limits (with InfluxDB Cloud)
+    #   
+    # For Rate limits, include a link to rate limits documentation for
+    # InfluxDB Cloud.
+    # If an endpoint introduces a change to limits or quotas, create a PR or 
+    # issue at https://github.com/influxdata/docs-v2/.
+    # 
+    # #### Related guides
+    # 
+    # For Related guides, include links to documented procedures and tutorials.
+    description: |
+      <!--- Example: Operation description for `POST /task`
+
+      Creates a task and returns the new task.
+
+      Use this endpoint to schedule a [Flux script]() that takes a stream of 
+      input data, modifies or analyzes it, and then writes the modified data
+      to a bucket or performs some action.
+
+      #### Required permissions
+
+      - `write-tasks`
+      - 'read-buckets'
+      - `write-buckets` or `write-bucket BUCKET_ID`
+      
+      `BUCKET_ID` is the ID of the destination bucket.
+
+      #### Limitations
+      
+      #### Rate limits (with InfluxDB Cloud)
+
+      `write` rate limits apply.
+      For more information, see [limits and adjustable quotas]({/influxdb/cloud/account-management/limits/).
```

Comment:
```suggestion
      For more information, see [limits and adjustable quotas]({{% INFLUXDB_DOCS_URL %}}/influxdb/cloud/account-management/limits/).
```

Reply:
wasn't sure if this link would work in context of this repo?

Reply:
oof